### ISSUE TITLE
SAN Support

### ISSUE THREAD

**Issue by NPPerkins** [lukesUbuntu/gogetsslv2#3]
Currently there's only support for single name certificates with the module, but I'd like to see if it'd be possible to get support for multi-domain certificates (SANs).

Reason being is that customers are usually searching for the best bang for their buck, and the multi-domain certificates offer the greatest bang for their buck if want to protect multiple domains. With this addition, it'd greater expand outreach to potential Blesta customers.

**Comment by lukesUbuntu** [comment]
I will look into implementing this, very good idea and will be useful.

**Comment by jamesm66** [comment]
:+1: I agree SAN support would be a great addition will you add this?

**Comment by lukesUbuntu** [comment]
Have started.

**Comment by NPPerkins** [comment]
I understand there's probably a million things going on, but was just wondering if there's any updates?

**Comment by lukesUbuntu** [comment]
@NPPerkins Yes sorry been flat out, im on holidays in 2 weeks so i will have a ton of time, i plan to push the new updates then.
